src/gui/virtualstudio.cpp

```diff
@@ -441,6 +441,24 @@ void VirtualStudio::setDebugText(QString text)
     emit debugTextChanged();
 }
 
+void VirtualStudio::joinStudio(const QUrl& url)
+{
+    if (url.scheme() != "jacktrip" || url.path().length() <= 1) {
+        return;
+    }
+    QString targetId = url.path().remove(0, 1);
```

Comment:
For what it's worth, the `path()` function turns
```
jacktrip://join/745646a0-704b-4ba9-895d-e5f6752dad08"
```
into
```
/745646a0-704b-4ba9-895d-e5f6752dad08
```
which is why there's the remove call following it